static/app/components/replays/breadcrumbs/replayTimelineEvents.tsx

```diff
@@ -40,7 +43,10 @@ function ReplayTimelineEvents({
         <EventColumn key={column} column={column}>
           <Event
             frames={colFrames}
+            handleMouseEnter={handleMouseEnter}
+            handleMouseLeave={handleMouseLeave}
```

Comment:
Should these props be `onMouse*`?

Reply:
makes sense

https://develop.sentry.dev/frontend/#event-handlers